inginious/frontend/templates/course_admin/task_list.html

```diff
@@ -113,4 +118,48 @@ <h5 class="modal-title" id="taskDispenserModalLabel">{{_("Switch task dispenser"
         </button>
     </div>
 </div>
+
+<script>
+	const tasksWithErrors = document.getElementsByClassName("bg-danger")
+
+	for(let task of tasksWithErrors){
+		document.getElementById("errorLogContainer").innerHTML += "<br>&nbsp &nbsp &nbsp<a href='#" + task.id + "' style='color: white; text-decoration: underline'>" + task.innerText + "</a><br>"
+	}
+
+	console.log(tasksWithErrors.length)
+
+	if(tasksWithErrors.length == 1){
+		document.getElementById("errorLog").innerText = "There is a task with some errors";
```

Comment:
Messages need to be localized. If you need these messages in JS, please export them from the jinja template using gettext. I'm honestly quite not sure this has to be done in javascript actually, as it is not really dynamic...